.github/workflows/test-coverage.yml

```diff
@@ -74,6 +74,12 @@ jobs:
       DEFRA_MUTATION_TYPE: ${{ matrix.mutation-type }}
 
     steps:
+      - name: Install ollama
+        run: curl -fsSL https://ollama.com/install.sh | sh
```

Comment:
question: This is only needed for basic test matrix?

Reply:
Yes. Why are you asking?

Reply:
because:
1) I don't understand why this is not needed for other test types (acp, view, encryption, etc.).
2) Is this a required dependency now? if so then:
	- document in `README`
	- add a make rule in `Makefile`
	- perhaps add this to the setup composite instead (`./.github/composites/setup-defradb`)

Reply:
1. When you see "Ollama", do you need more documentation to know what this is about? I might be wrong to assume that would be enough to understand that this is only for embedding related testing.
2. It's not a Defra dependency. Only a testing dependency for the embedding specific tests.

Reply:
1) Maybe I missed it but last I checked the embedding tests they didn't have `SupportedTypes` like config. So I assumed they will run with all configurations. For example when encryption or macos tests run, wouldn't they still run and hence be need this dep?

Reply:
2. Yea so similar to how we have rust dep for lens test, this should have a make rule imo and will be part of test/dev deps